nomic/dataset.py

```diff
@@ -1160,21 +1167,11 @@ def create_index(
                 "model_hyperparameters": None,
                 "nearest_neighbor_index": "HNSWIndex",
                 "nearest_neighbor_index_hyperparameters": json.dumps({"space": "l2", "ef_construction": 100, "M": 16}),
-                "projection": "NomicProject",
-                "projection_hyperparameters": json.dumps(
-                    {
-                        "n_neighbors": projection.n_neighbors,
-                        "n_epochs": projection.n_epochs,
-                        "spread": projection.spread,
-                        "local_neighborhood_size": projection.local_neighborhood_size,
-                        "rho": projection.rho,
-                        "model": projection.model,
-                    }
-                ),
+                "projection_hyperparameters": json.dumps(projection_hyperparameters),
```

Comment:
I think we can always use "NomicProject", it isn't influencing anything (it will also fix the unit tests)